docs/arvo/subject.md

```diff
@@ -58,8 +58,8 @@ Hoon, we call the code executed the "formula" and its context the
       (add b $(b (add b 5)))    ::
                                 ::
     ++  sum                     ::
-      |=  a/@u                  ::
-      (add (five a) (three a))  ::
+      |=  {a/@u b/@u}           ::
+      (add (five a) (three b))  ::
```

Comment:
`++sum` should probably be just the single-argument version both here and where it's called.  I don't think there's any pedagogical reason to use the two-argument version?